app/models/katello/repository.rb

```diff
@@ -205,6 +205,17 @@ def self.in_content_views(views)
         .where("#{Katello::ContentViewVersion.table_name}.content_view_id" => views.map(&:id))
     end
 
+    def self.feed_ca_cert(url)
+      file = feed_ca_file(url)
+      File.read(file) if file
+    end
+
+    def self.feed_ca_file(url)
+      if Katello::Resources::CDN::CdnResource.redhat_cdn?(url)
+        ::Katello::Resources::CDN::CdnResource.ca_file
+      end
```

Comment:
Any reason this can't be one line? Also - very nitpicky -  but you have a top level `::` on one name class, but not the other